frontend/setupTests.ts

```diff
@@ -2,9 +2,18 @@
 import '@testing-library/jest-dom/vitest';
 
 import * as matchers from '@testing-library/jest-dom/matchers';
-import { config } from 'dotenv';
+import dotenv from 'dotenv';
 import { expect, afterEach, beforeAll, afterAll } from 'vitest';
-config();
+
+let envFile = '.env.dev';
+
+if (process.env.NODE_ENV === 'production') {
+  envFile = '.env.prod';
+} else if (process.env.NODE_ENV === 'test') {
+  envFile = '.env.dev';
+}
+
+dotenv.config({ path: envFile });
```

Comment:
p4: 환경변수 설정 좋은데요~ let을 안쓰고 이렇게도 쓸 수 있어서 말해봤어!

```suggestion
import { config } from 'dotenv';
import { expect, afterEach, beforeAll, afterAll } from 'vitest';

if (process.env.NODE_ENV === 'production') {
  config({ path: '.env.prod' }); 
} else if (process.env.NODE_ENV === 'test') {
  config({ path: '.env.dev' }); 
}

```

Reply:
수정커밋: [refactor: dotenv 환경 변수 설정에서 let 제거 및 간결화](https://github.com/woowacourse-teams/2025-Fit-toring/pull/597/commits/afb95b13b780531284eebb1181fdace452c78ea3)